package/libskylark/tests/test_libskylark.cc

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2017 Swift Navigation Inc.
+ * Contact: Swift Navigation <dev@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#include "gtest/gtest.h"
+
+extern "C" {
+#include "libskylark.h"
+}
+
+#define DEFAULT_DEVICE_UID      "22222222-2222-2222-2222-222222222222"
```

Comment:
What's the expected way that this is written to "/cfg/device_uuid" on the test machine?